aries-site/src/examples/components/table/TableSingleSelectExample.js

```diff
@@ -193,27 +201,55 @@ const columns = [
   },
 ];
 
-const onClickHandler = record => {
-  // eslint-disable-next-line no-alert
-  alert(`
-    Record was clicked:
-    { 
-        id: ${record.id},
-        orderName: ${record.orderName}
-        state: ${record.state}
-        orderDate: ${record.orderDate}
-    }
-    
-    You can use onClick() to navigate to a record's detail
-    page, open a panel or modal to edit the record, or perform 
-    other actions as you see fit.
-  `);
-};
-
 export const TableSingleSelectExample = () => {
   const size = React.useContext(ResponsiveContext);
+  const [pageDetails, setPageDetals] = React.useState('');
 
-  return (
+  return pageDetails.id !== undefined ? (
```

Comment:
For clarity for those coming into this, can we swap the condition so the Table return comes first?

`!showPageDetails ? <Table /> : <PageDetails />`